automated install/basic-install.sh

```diff
@@ -144,17 +144,20 @@ setcap_check(){
   # create test file
   ${SUDO} touch /usr/bin/pihole.setcap.test
   # set the file with the correct capabilities and awk possible failure
-  setcap_status="$(${SUDO} setcap CAP_NET_BIND_SERVICE,CAP_NET_RAW,CAP_NET_ADMIN+eip /usr/bin/pihole.setcap.test 2>&1 | awk 'FNR == 1 {print $1}')"
+  setcap_status="$(${SUDO} setcap CAP_NET_BIND_SERVICE,CAP_NET_RAW,CAP_NET_ADMIN+eip /usr/bin/pihole.setcap.test 2>&1 | awk 'FNR == 1')"
```

Comment:
I don't think you need the awk command anymore?

Reply:
setcap spits out 2 or 3 lines of information when facing an error.  Only the first line is the actual error message and the rest is general guides.

The extra (irrelevant)  information is either 

"The value of the capability argument is not permitted for a file. Or the file is not a regular (non-symlink) file"

Or
usage: setcap [-q] [-v] (-r|-|<caps>) <filename> [ ... (-r|-|<capsN>) <filenameN> ]

 Note <filename> must be a regular (non-symlink) file.

awk on the first line is used to return the actual error message of setcap, before stopping the install.